Makefile

```diff
@@ -1,38 +1,42 @@
 .DEFAULT_GOAL := help
 .PHONY: help build test
 
+PYPI_TOKEN := $(shell echo ${PYPI_TOKEN})
```

Comment:
Could you please replace `PYPI_TOKEN` with `PYPI_PASSWORD` since it's already configured in the repo? same for test one